index.d.ts

```diff
@@ -39,7 +39,25 @@ declare class AutoNumeric {
     constructor(
         element: string | HTMLElement,
         initialValue?: string | number | null,
-        options?: CallbackOptions | string | null
+        options?: CallbackOptions | string | (CallbackOptions | string)[] | null
+    );
+    /**
+     * Enables the auto numeric feature for the given element.
+     * 
+     * The DOM element must be one of the allowed elements:
+     * 
+     * > b, caption, cite, code, const, dd, del, div, dfn, dt, em, h1, h2, h3, h4, h5, h6, input, ins, kdb, label, li, option, output, p, q, s, sample, span, strong, td, th, u
```

Comment:
Please refer to `AutoNumericEnum.allowedTagList`, since this list can change and we do not want to have to update it in 2 places

Reply:
Since the AutoNumericEnum object does not seems to be exposed, and the `AutoNumeric.prototype.tagList` property does appear to be intended to be part of the public API either (?), I just removed the list of tags and replaced it with a general comment referring to the documentation.